5_databases/week1/Homework1

```diff
@@ -0,0 +1,47 @@
+-- Active: 1706445451493@@127.0.0.1@3306@HYFSql
```

Comment:
Nice job Angel. Can you please rename the file to Homework1.sql instead of Homework1? that way github would know it's a sql file and you are writing sql queries :)

Reply:
Thanks for the review Mahmoud... I have already made the correction